charts/rd-commondata-api/values.yaml

```diff
@@ -24,7 +24,7 @@ java:
           alias: azure.application-insights.instrumentation-key
         - name: LD-SDK-KEY
           alias: LD_SDK_KEY
-        # - name: commondata-api-POSTGRES-PASS
-        #   alias: spring.datasource.password
+        - name: commondata-api-POSTGRES-PASS
+          alias: spring.datasource.password
```

Comment:
you can use this alias `POSTGRES_PASSWORD`

Reply:
Sorry, you need to use ` alias: POSTGRES_PASSWORD` instead of `alias: spring.datasource.password`